src/server/helpers/token.js

```diff
@@ -0,0 +1,38 @@
+import jwt from 'jsonwebtoken';
+import crypto from 'crypto';
+import { response } from '../helpers/errorHandler';
+
+const { SECRET = '669141b534e06cf5c0271a389b79ebc11910f86bd210a74b6f00fcb974010809' } = process.env;
```

Comment:
Don’t use smth secret, if it can be seen 🙂

Reply:
done

Reply:
i used for it dotenv package, and all future env consts will in .env